### DISCUSSION TITLE
can't use this app on nixos

### DISCUSSION THREAD

**Discussion by cStralpt** [anyproto/contributors#202] [Q&A]
i've download the latest version of anytype and it's didn't work
![image](https://github.com/anyproto/contributors/assets/95400822/1e126778-9e31-4b02-8a81-4cb9237fec25)

**Comment by fb929** [comment]
You need to install the "libsecret" library:
https://discourse.nixos.org/t/problem-running-appimage/10094/2

**Comment by hellodword** [comment]
Use the package in nixpkgs: https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name/an/anytype

Or nur: https://github.com/nix-community/nur-combined/tree/master/repos/kira-bruneau/pkgs/development/tools/misc/anytype

Or a flake and build from source: https://github.com/squalus/anytype-flake
